src/observer.tsx

```diff
@@ -40,7 +40,7 @@ export function observer(component: IReactComponent<any>): IReactComponent<any>
             throw new Error("render property of ForwardRef was not a function")
         return React.forwardRef(function ObserverForwardRef() {
             return <Observer>{() => baseRender.apply(undefined, arguments)}</Observer>
-        })
+        }) as any
```

Comment:
Perhaps we might include `RefForwardingComponent` in `IReactComponent` as well? Given that we actually try to identify such a component, it should be supported to be passed in.

Reply:
Sounds good. Also replaced Stateless with Functional in next commit.